app/scripts/components/facets/facets.controllers.ts

```diff
@@ -98,6 +99,10 @@ module ngApp.components.facets.controllers {
       this.FacetService.addTerm(facet, term);
     }
 
+    addMissing(facet: string): void {
```

Comment:
why do you need an addMissing function? `_missing` should just work like any other term shouldn't it?